docker-compose.test.yml

```diff
@@ -0,0 +1,6 @@
+sut:
+    build: .
```

Comment:
# `build` ignores the cache in Auto Test!

When doing any docker changes (especially due to the `COPY` command, see PR #12) it would take **one hour** to Auto Build the image, then **another hour** to build it again for Auto Test. Even if the Auto Build would take a few seconds thanks to the cache it would take one hour to build the image for Auto Test. 🤯

`build: .` was changed to `image: seriema/retro-cloud:${CACHE_TAG}` in 37e8650.
Reference: https://forums.docker.com/t/auto-test-rebuilds-image/68481/6

It's quite easy to find the fix in this graph 😁 (The two big bumps at the end are rebuilds of the master branch that don't have the fix yet)
![image](https://user-images.githubusercontent.com/693684/79047512-4f9cf580-7c17-11ea-9da2-86fc35424d3f.png)